src/services/Example.service.ts

```diff
@@ -1,17 +1,23 @@
 import { Injectable } from "@nestjs/common";
```

Comment:
Build failling due to Capital file name Example.service

Reply:
ahh thanks!